src/components/marquee-wizard/marquee-wizard.component.ts

```diff
@@ -89,6 +92,17 @@ export class MarqueeWizardComponent extends WizardComponent implements OnDestroy
         }
     }
 
+
+    /**
+     * If the validation is true then disable the next button
+     * and do not allow progression onto the next step
+     */
+    stepDisabled(): boolean {
```

Comment:
Perhaps `isNextDisabled()` might be a more self explanatory name, just to clarify it is the next button that is being checked not the step?

Also could you review the whitespace here, we could remove the empty line and the double space after the `&&`.

Consider adding the following to your VS Code settings:

![image](https://user-images.githubusercontent.com/20795331/72518301-efcbc400-384c-11ea-9f0a-56449e829a51.png)

![image](https://user-images.githubusercontent.com/20795331/72518331-0114d080-384d-11ea-871a-9950129a3a18.png)

```json
"[typescript]": {
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": true
        },
        "editor.defaultFormatter": "vscode.typescript-language-features"
    },
```

This will automatically format your code correctly when you save a file.